recipes/rdkit/bld.bat

```diff
@@ -0,0 +1,47 @@
+rem surely there's a better way than this
+if "%PY_VER%"=="2.7" (
+	set PYTHON_LIBRARY=python27.lib
+) else if  "%PY_VER%"=="3.4" (
+	set PYTHON_LIBRARY=python34.lib
+) else if  "%PY_VER%"=="3.5" (
+	set PYTHON_LIBRARY=python35.lib
+) else if  "%PY_VER%"=="3.6" (
+	set PYTHON_LIBRARY=python36.lib
+) else (
+	echo "Unexpected version of python"
+	exit 1
+)
+
+where jom 2> NUL
+if %ERRORLEVEL% equ 0 (
+  set MAKE_CMD=jom -j%CPU_COUNT%
```

Comment:
So instead of checking if `jom` is available, we can just require `jom` as there is a `conda-forge` package for it.